data/js/about-newtab.js

```diff
@@ -2,19 +2,30 @@
 
 var headDiv = document.querySelector('#newtab-search-container');
 var footDiv = document.querySelector('#newtab-margin-bottom');
-var undoMigrate = document.querySelector('#undo-migrate');
 
 function showUserData(headerContent, footerContent) {
     headDiv.insertAdjacentHTML('beforebegin', headerContent);
     footDiv.insertAdjacentHTML('beforebegin', footerContent);
+
+    /* staging for auto migrate
+    var undoMigrate = document.querySelector('#undo-migrate');
+
+    undoMigrate.addEventListener('click', function() {
+	    addon.port.emit('undoMigrate');
+	});*/
```

Comment:
Nit: Even though this is commented out at the moment, just double check your indentation.